doc/specs/vulkan/chapters/devsandqueues.txt

```diff
@@ -347,6 +347,13 @@ include::../structs/VkDeviceCreateInfo.txt[]
 
 include::../validity/structs/VkDeviceCreateInfo.txt[]
 
+fname:vkCreateDevice creates a VkDevice object, then enables
+extensions requested by the application. If an extension
+is provided by a layer, the layer must: be specified at
```

Comment:
oook then, so it **is** a requirement, so it is not (can't be) handeled by VK_ERROR_EXTENSION_NOT_PRESENT return.

But in that case should be caught by validation layers (and can be easily so, I think) and as such that would be good inside VU. (IIRC statements was removed from VU because of other things, not because of this).